vm/src/pyobject.rs

```diff
@@ -1259,6 +1259,26 @@ where
     }
 }
 
+pub trait IntoPyClass {
```

Comment:
If the macro generates an implementation of `IntoPyClass`, then shouldn't it be a derive mode macro?

Reply:
No, because you can't `#[derive()]` on an `impl` block.